.github/workflows/publish.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         run: |
           sudo apt-get -y remove libzmq* || true
           sudo apt-get -y install software-properties-common gcc-9 g++-9 || true
+          sudo apt-get -y install liblz4-dev || true
```

Comment:
maybe we can add this library to our deps script instead of installing in through apt-get? seems like it is pretty easy to install https://github.com/lz4/lz4